### DISCUSSION TITLE
Community update blog post

### DISCUSSION THREAD

**Discussion by Daeraxa** [pulsar-edit/.github#122] [Announcements]
Check out the first in what we hope will be our regular updates on the [Pulsar Blog](https://pulsar-edit.dev/blog/20230201-Daeraxa-FebUpdate.html)!

This will hopefully give you some insight as to what we have been up to in the background and what you can expect expect to see in the near future.

In this update we have info on tree-sitter upgrades, a new donation platform, backend improvements and an upcoming Matrix space.

**Comment by confused-Techie** [comment]
Love to see it! Also love trying to reach users not active on Discord here!
